examples/hcp-ec2-demo/main.tf

```diff
@@ -21,7 +21,7 @@ resource "hcp_hvn" "main" {
 }
 
 module "aws_hcp_consul" {
-  source = "hashicorp/hcp-consul/aws"
+  source = "../../../terraform-aws-hcp-consul"
```

Comment:
These sources should stay pointed at the remote registry, so that people can easily copy and paste them.